src/Uno.UI.Runtime.Skia.Wpf/Uno.UI.Runtime.Skia.Wpf.csproj

```diff
@@ -19,7 +19,7 @@
 		<PackageId Condition="'$(UNO_UWP_BUILD)'!='true'">Uno.WinUI.Runtime.Skia.Wpf</PackageId>
 	
 		<UseCommonOverridePackage>true</UseCommonOverridePackage>
-		<UseWPF Condition="'$(TargetFramework)'=='netcoreapp3.1' or '$(TargetFramework)'=='net5.0-windows'">true</UseWPF>
+		<UseWPF Condition="'$(TargetFramework)'=='netcoreapp3.1' or '$(TargetFramework)'=='net5.0-windows' or '$(TargetFramework)'=='net47'">true</UseWPF>
```

Comment:
I wonder why this condition exists? Why not an always true? cc @jeromelaban

Reply:
@jeromelaban added this when he [added support for .NET Core](https://github.com/unoplatform/uno/commit/e2d4eb209a5dece76eed28b0c01e8956ffe9f403). I don't know if he means "Use the SDK's WPF if it's .NET Core, else use reference assemblies", or if he means to build for other frameworks in the future.

If he actually meant the former, I'll remove this line all together.

Reply:
This property is supposed to be only useful when building for netcoreapp, not for net47. It seems that it's used by `MSBuildSDK.Extras` but it may not make sense in a net47 scenario.

Reply:
This branch has passed the nuget build tests, I guess this should work.